src/main/webapp/app/shared/icons/CommentIcon.tsx

```diff
@@ -58,6 +67,45 @@ const CommentIcon = observer((props: ICommentIconProps) => {
     timeoutId.current = id;
   }
 
+  async function handleCreateComment(content: string) {
+    // replace with runTransaction?
```

Comment:
runTransaction is meaningful when there are multiple data elements are involved. I don't think you need one for this method.